src/api/dps.js

```diff
@@ -0,0 +1,34 @@
+import axios from "axios";
+
+class DevProgramClient {
+  constructor(token) {
+    this.instance = axios.create({
+      baseURL: process.env.NEXT_PUBLIC_DEV_PLATFORM_URL,
+      headers: {
+        Authorization: `Bearer ${token}`,
+        "Content-Type": "application/json",
+      },
+    });
+  }
+
+  async getAllBookmarks() {
+    try {
+      const res = await this.instance.get(`/v2/links`);
+      return res.data;
+    } catch (error) {
+      console.log(error);
+      throw new Error(`Can't get bookmarks [${error.message}]`);
```

Comment:
First of all, I'd keep error messages consistent with our other projects. Besides that, it makes sense only if we display them somewhere. For example, by using our notification system.